src/demo-app/a11y/table/table-a11y.html

```diff
@@ -0,0 +1,70 @@
+<div>
+  <section>
+    <h2>Basic Table</h2>
+    <p>Shows name, color and age data.</p>
+    <md-table [dataSource]="basicDataSource">
+      <ng-container cdkColumnDef="name">
+        <md-header-cell *cdkHeaderCellDef>Name</md-header-cell>
+        <md-cell *cdkCellDef="let row">{{row.name}}</md-cell>
+      </ng-container>
+      <ng-container cdkColumnDef="color">
+        <md-header-cell *cdkHeaderCellDef>Color</md-header-cell>
+        <md-cell *cdkCellDef="let row" [style.color]="row.color">{{row.color}}</md-cell>
```

Comment:
Some colors are hard to read like yellow. Can we make this more accessible? We can remove the styled colors